scripts/prep.sh

```diff
@@ -24,4 +24,9 @@ docker pull rancher/k3d-tools:5.0.0
 docker pull python:3.7-slim
 
 section "Preparing Filesystem..."
+
+if [ -d "/tmp/src" ]; then
+  rm -rf /tmp/src
+fi
+
 mkdir /tmp/src
```

Comment:
Just make `mkdir` not yield an error instead :)

```suggestion
if [ -d "/tmp/src" ]; then
  rm -rf /tmp/src
fi

mkdir -p /tmp/src
```